src/include/maat/process.hpp

```diff
@@ -13,11 +13,13 @@ class ProcessInfo: public serial::Serializable
 {
 public:
     int pid; ///< Process ID  
+    int ruid; ///< Real user ID
+    int euid; ///< Effective user ID
     std::string pwd; ///< Current working directory of the running process
     std::string binary_path; ///< Path to the executable in the virtual file system
     bool terminated; ///< 'True' if the process exited or was killed
 public:
-    ProcessInfo(): pid(0), pwd(""), binary_path(""), terminated(false){}
+    ProcessInfo(): pid(0), ruid(500), euid(500), pwd(""), binary_path(""), terminated(false){}
```

Comment:
I'd be in favour of initialising `ruid` and `euid` to zero by default. They should receive their proper value (`500` is fine) at the same place where `pid` does, which is here: 

https://github.com/trailofbits/maat/blob/edd4e71dc9bfb3e0231f595971ce850947c4f900/src/loader/loader_lief.cpp#L41-L42